src/App/Templates/content/Forms/constants.js

```diff
@@ -0,0 +1,135 @@
+import InputGroup from "@components/InputGroup";
+import Links from "@components/Link";
+import ProgressSteps from "@components/Steps";
+import RadioButton from "@components/FormComponents/Radio";
+
+const steps = [
+    {
+        title: "Step one",
+        subtitle: "Authentication",
+        completed: true
+
+    },
+    {
+        title: "Step two",
+        ongoing: true,
+        selected: true
+    },
+    {
+        title: "Step three"
+    },
+    {
+        title: "Step four"
+    }
+];
+
+const radioOptions = [
+    {
+        label: "Option 1",
+        id: "options-1",
+        checked: true
+    },
+    {
+        label: "Option 2",
+        id: "options-2"
+    },
+    {
+        label: "Option 3",
+        id: "options-3"
+    }
+];
+
+export const formsShowCase = {
+    id: "overview-forms",
+    tabsId: "overview-forms-tabs",
+    hideOptions: true,
+    elements: [
+        {
+            tab: "Form",
+            component:
+            <div className="col-md-10 col-12 px-0">
+                <form>
+                    <fieldset>
+                        <legend>
+                            <span className="h2">Contact us</span>
+                        </legend>
+                        <div className="col-12 col-sm-9 px-0">
+                            <InputGroup type="email" label="E-mail"/>
+                            <InputGroup type="select" label="Subject" placeholder="Select a subject" optional selectOptions={["Subject 1", "Subject 2", "Subject 3"]}/>
+                        </div>
+                        <InputGroup type="textarea" label="Message" />
+                        <Checkbox
+                            id="forms-template-checkbox-example"
+                            label={<>I have read and accept Swedbank Pays&#39;s&nbsp;<Links linkText="privacy policy" rightIcon="launch" /></>} />{"\n"}
+                        <Button type="primary" label="Send message"/>
+                    </fieldset>
+                </form>
+            </div>,
+            options: {
+                checkbox: [{ inputs: [{}] }]
+            },
+            title: "Form",
+            description: "This in an example of a single page contact form using a collection of components including input fields, select list, checkbox and button."
+        },
+        {
+            tab: "Multi-step form",
+            component:
+            <div className="col-md-10 col-12 px-0">
+                <form>
+                    <ProgressSteps steps={steps} />
+                    <fieldset className="">
+                        <legend>
+                            <span className="h2">Step two</span>
+                        </legend>
+                        <div className="d-flex justify-content-between">
+                            <InputGroup type="text" label="Label" className="w-50 mr-1"/>
+                            <InputGroup type="text" label="Label" className="w-50 ml-1"/>
+                        </div>
```

Comment:
Not sure if this will work on mobile, use ``row` and `col-12 col-sm-6` on the input groups